2019-06-19-all-hands-meeting.md

```diff
@@ -0,0 +1,112 @@
+# 2019-06-19 Toronto Workers Co-op: All-Hands Meeting
+
+<sup>[from template][template]</sup>
+
+[:date: Calendar][cal] | [:cat: GitHub][gh] | [:open_file_folder: GDrive][gdrive] | [:notebook: Meeting Index][meetings]
+
+Time: Wed, June 12, 2019 @ 5-6pm ET  
+Location: https://meet.jit.si/hyphacoop  
+Attending: dcwalk, yurko, elon, patcon, ben
+
+Notetaker: elon
+
+:raising_hand: Next: patcon, udit, dc, elon, yurko, garry, benedict  
+<sup>Bump to end on taking turn :)</sup>
+
+[TOC]
+
+## Agenda
+
+- Previous Action Items
+    - [x] patcon: update [template][template] with new call url
+    - [x] patcon: update template with new format ([example](https://github.com/hyphacoop/organizing/blob/master/2019-05-15-all-hands-meeting.md))
+    - [x] patcon, garry: meet to discuss Admin PM & task tracking
+    - [ ] elon: organizing larger infra strategy WG meetings
```

Comment:
```suggestion
    - [x] elon: organizing larger infra strategy WG meetings
```

Set for friday :) @ASoTNetworks, I think it still needs to go in calendar, but I'm guessing this item is closed once that's done?

Reply:
Also, should we discuss/consense/doc what's involved in "scheduling a meeting"? We just kinda have it undefined, and I feel for it being hard to know what's done vs done-done :) happy to leave it fuzzy though, if that's the pref

Reply:
Re-ticketed so we don't lose or get stuck here in this code review :) https://github.com/hyphacoop/handbook/issues/10